types-doc/oci-definition.md

```diff
@@ -44,7 +44,7 @@ The structure of a PURL for this package type is:
 - `pkg:oci/debian@sha256%3A244fd47e07d10?repository_url=docker.io/library/debian&arch=amd64&tag=latest`
 - `pkg:oci/debian@sha256%3A244fd47e07d10?repository_url=ghcr.io/debian&tag=bullseye`
 - `pkg:oci/static@sha256%3A244fd47e07d10?repository_url=gcr.io/distroless/static&tag=latest`
-- `pkg:oci/hello-wasm@sha256%3A244fd47e07d10?tag=v1`
+- `pkg:oci/hello-wasm@sha256:A244fd47e07d10?tag=v1`
 
```

Comment:
This looks wrong to me. According to the spec, the version string MUST be percent encoded; And furthermore, the edit doesn't even correctly remove the encoding.

Reply:
@sjn  eagle eyes! :eagle: ... the colon should never be encoded anywhere per the spec, but the changes should be this as `%3A` is the encoded colon:
```suggestion
- `pkg:oci/hello-wasm@sha256:244fd47e07d10?tag=v1`
```

@TG1999 ^

Reply:
@pombredanne ack! @sjn good cath \o/